manifests/windows/adserver.pp

```diff
@@ -91,20 +91,17 @@
     ensure => directory,
   }
 
-  class { 'staging':
-    path    => 'C:/shares',
-  }
-  staging::file { 'Brackets.msi':
+  archive { 'C:/shares/Brackets.msi':
```

Comment:
@binford2k Can't remember for sure, but does the lab require the student to install from `\\ADSERVER\installer\Brackets.msi`? If so, does the MSI file need to be downloaded into `C:/shares/classroom` on the AD server instead of `C:/shares`?

Reply:
I've got no idea, to be honest. I was just porting straight over. But I'll go check.

Reply:
Sure enough, this appears to have been broken for a while. Thanks for the catch.

```
12:09 $ grep -rn 'Brackets.msi' windowsessentials/
windowsessentials//Packages_and_Services/Lab-unc_package.md:46:    * Set the `source` to `'//ADSERVER/installer/Brackets.msi'`
windowsessentials//_files/win_essentials/solutions/brackets.pp:5:    source          => '//ADSERVER/installer/Brackets.msi',
```